kakao/src/main/kotlin/com/agoda/kakao/searchview/SearchViewActions.kt

```diff
@@ -0,0 +1,37 @@
+package com.agoda.kakao.searchview
+
+import android.view.View
+import android.widget.SearchView
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers
+import com.agoda.kakao.common.actions.BaseActions
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+
+interface SearchViewActions : BaseActions {
+
+	fun typeQuery(query: String, append: Boolean = false) {
```

Comment:
Providing `boolean` as argument to function actually not a clear way. _(Clean code - Robert Martin )_

Could you please separate logic for two different functions for single responsibility purpose such as

```
fun typeQuery
fun appendQuery
```